.github/workflows/build.yml

```diff
@@ -66,6 +66,18 @@ jobs:
       with:
         path: 'examples'
 
+  build-release-v4_4:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v2
+      with:
+        submodules: 'recursive'
+    - name: esp-idf build
+      uses: espressif/esp-idf-ci-action@release-v4.4
+      with:
+        path: 'examples'
+
```

Comment:
I think you can remove build-release-v3_3 and build-release-v4_0, since these two IDF releases are now end-of-life. This will allow us to simplify the CMakeLists file!